### PR TITLE
RSDK-5313 remove extra from slam GetPositionResponse 

### PR DESCRIPTION
**Pull request by oliviamiller**

### CODE REVIEW

**Review by martha-johnston — APPROVED** [review]
lgtm pending full scope approval

**Comment by martha-johnston** [comment]
fyi just made this change locally to test go changes and had to remove line 7 because it's unused when you run `make dist/buf` to build the protos

**Comment by oliviamiller** [comment]
> fyi just made this change locally to test go changes and had to remove line 7 because it's unused when you run `make dist/buf` to build the protos

thanks for letting me know just removed it!

**Pull request merged**
